src/cpp/modules/tapenade/gmm/gmm.cpp

```diff
@@ -133,15 +148,16 @@ void Qtimesx(
     double* out
 )
 {
-    for (int id = 0; id < d; id++)
+    int i, j;
```

Comment:
Prefer to declare in as local a scope as possible
```
for(int i … ) 
 for(int j ….)
```

Reply:
I have tried to create .cpp files for functions as C-like as possible because Tapenade can differentiate only C and Fortran code. Thus we can see how Tapenade generates code of a derivative from the given source (except errors which are marked by comments). So, I think it would be better not to change these parts. Or am I wrong? What do you think about my argument?

Reply:
Problem was discussed in a meeting. Resolved.